### PR TITLE
Update resources

### PR DESCRIPTION
**Pull request by filippolmt**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -25,8 +25,10 @@ module "gke" {
   maintenance_start_time          = var.gke_maintenance_start_time
   maintenance_end_time            = var.gke_maintenance_end_time
   maintenance_recurrence          = var.gke_maintenance_recurrence
-  datapath_provider               = var.gke_datapath_provider
```
Comment (paolomainardi) [inline main.tf:28]: why are we removing this ?

File: README.md [hunk 2]
```diff
@@ -38,31 +40,31 @@ For a total of:
 - GKE Autopilot free-tier: **65$/monthly**
 - GKE Autopilot: **135$/monthly**
 
-> NOTE: Elasticsearch integration is not yet finished.
+> TODO: Elasticsearch integration is not yet integrated here.
```
Comment (paolomainardi) [inline README.md:43]: we can remove this comment, it is integrated now.

File: main.tf (hunk 1 above)
Comment (paolomainardi) [reply main.tf:28]: ok thanks

File: README.md (hunk 2 above)
Comment (filippolmt) [reply README.md]: Done

**Pull request merged**